### PR TITLE
fix source URL for SUNDIALS 5.x

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
follow-up for #14092

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/s/SUNDIALS/SUNDIALS-5.7.0-foss-2020b.eb [hunk 1]
```diff
@@ -10,9 +10,9 @@ description = "SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation
 toolchain = {'name': 'foss', 'version': '2020b'}
 toolchainopts = {'openmp': True, 'usempi': True, 'pic': True}
 
-source_urls = ['https://computation.llnl.gov/projects/sundials/download/']
+source_urls = ['https://github.com/LLNL/sundials/releases/download/v%(version)s/']
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['8d6dd094feccbb8d6ecc41340ec16a65fabac82ed4415023f6d7c1c2390ea2f3']
+checksums = ['48da7baa8152ddb22aed1b02d82d1dbb4fbfea22acf67634011aa0303a100a43']
```
Comment (boegel) [inline easybuild/easyconfigs/s/SUNDIALS/SUNDIALS-5.7.0-foss-2020b.eb:15]: Huh, that's clearly wrong, not sure how this slipped in... Fixing with a force push.

Comment (boegel) [reply easybuild/easyconfigs/s/SUNDIALS/SUNDIALS-5.7.0-foss-2020b.eb:15]: Actually, no, it's the correct checksum when downloading from GitHub, so the source tarball is not identical to the original one... 🤦

**Comment by boegel** [comment]
Diff between old and new source tarball for SUNDIALS 5.7.0:

```
diff -ru sundials-5.7.0.old sundials-5.7.0
Binary files sundials-5.7.0.old/doc/arkode/ark_examples.pdf and sundials-5.7.0/doc/arkode/ark_examples.pdf differ
Binary files sundials-5.7.0.old/doc/arkode/ark_guide.pdf and sundials-5.7.0/doc/arkode/ark_guide.pdf differ
Binary files sundials-5.7.0.old/doc/cvode/cv_examples.pdf and sundials-5.7.0/doc/cvode/cv_examples.pdf differ
Binary files sundials-5.7.0.old/doc/cvode/cv_guide.pdf and sundials-5.7.0/doc/cvode/cv_guide.pdf differ
Binary files sundials-5.7.0.old/doc/cvodes/cvs_examples.pdf and sundials-5.7.0/doc/cvodes/cvs_examples.pdf differ
Binary files sundials-5.7.0.old/doc/cvodes/cvs_guide.pdf and sundials-5.7.0/doc/cvodes/cvs_guide.pdf differ
Binary files sundials-5.7.0.old/doc/ida/ida_examples.pdf and sundials-5.7.0/doc/ida/ida_examples.pdf differ
Binary files sundials-5.7.0.old/doc/ida/ida_guide.pdf and sundials-5.7.0/doc/ida/ida_guide.pdf differ
Binary files sundials-5.7.0.old/doc/idas/idas_examples.pdf and sundials-5.7.0/doc/idas/idas_examples.pdf differ
Binary files sundials-5.7.0.old/doc/idas/idas_guide.pdf and sundials-5.7.0/doc/idas/idas_guide.pdf differ
Binary files sundials-5.7.0.old/doc/kinsol/kin_examples.pdf and sundials-5.7.0/doc/kinsol/kin_examples.pdf differ
Binary files sundials-5.7.0.old/doc/kinsol/kin_guide.pdf and sundials-5.7.0/doc/kinsol/kin_guide.pdf differ
```

That looks harmless enough to consider the checksums for both the old and new source tarballs valid...

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--force-download --sourcepath /tmp/$USER/pr14096"

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @boegel!

**Pull request merged**